content/resources/mat102.md

```diff
@@ -40,3 +56,38 @@ desc="A podcast where mathematicians talk about their life and research"></grid-
 img-Src="https://images.squarespace-cdn.com/content/5436e695e4b07f1e91b30155/1413407411253-0L77UY94Y6UDHOO3AD6E/?content-type=image%2Fjpeg"
 button="Start Listening!"
 desc="The acclaimed mathematician and author Steven Strogatz interviews some of the world’s leading scientists about their lives and work."></grid-1-x-2>
+
+## Resources
+
+<grid-1-x-2
+title="Intro to Proofs"
+link='https://www.youtube.com/watch?v=V5tUc-J124s'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.6SJaapSUZ8k6Levep82aJgAAAA%26pid%3DApi&f=1"
+button="Start Listening!"
+desc="This is an introduction to proofs, introducing techniques likes Contradiction, Contrapositive, etc."></grid-1-x-2>
+
+### Field Axioms
+
+<iframe width="500" height="400" src="https://www.youtube.com/embed/ndniuygmXEo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<grid-1-x-2
+:reversed=true
+title="Mathematical Induction"
+link='https://www.youtube.com/watch?v=ndniuygmXEo'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.cnNMXOx79EDHDbAFOEopzQHaDt%26pid%3DApi&f=1"
+button="Start Listening!"
+desc="This video will introduce you to mathematical induction."></grid-1-x-2>
+
+<grid-1-x-2
+title="Euclidean geometry"
+link='https://www.youtube.com/watch?v=wQAiytWGgEk'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.aj6WCdmPleUSyez68D7PIQHaB3%26pid%3DApi&f=1"
+button="Check it out!"
+desc="This video is an introduction to Euclidean geometry a major part of MAT102."></grid-1-x-2>
+
+## What courses are avaliable after MAT102:
+
+### CS POSt
```

Comment:
Perhaps, we could move CS POSt to another resource page rather than having it as a "course" available after MAT102